app/Http/Transformers/ReactionTransformer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Reaction;
+use League\Fractal\TransformerAbstract;
+
+class ReactionTransformer extends TransformerAbstract
+{
+    /**
+     * Transform the resource data.
+     *
+     * @param \Rogue\Models\Reaction $reaction
+     * @return array
+     */
+    public function transform(Reaction $reaction)
+    {
+        return [
+            'reaction_id' => $reaction->id,
```

Comment:
I think we want to cast `reaction_id` and `reportback_item_id` as `strings` in the transformer. Since the id represents a record not something you would do math on, so it isn't really an `int`. Same thing we did in the transformer [here](https://github.com/DoSomething/rogue/blob/master/app/Http/Transformers/ReportbackTransformer.php#L28)